package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "egg-path-matching",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Don't modify version

Reply:
revert this version change and ok to be merge. @xyeric

Reply:
done, thx~